src/PointwiseFunctions/MathFunctions/Gaussian.hpp

```diff
@@ -85,12 +111,123 @@ class Gaussian : public MathFunction<1> {
   T apply_second_deriv(const T& x) const noexcept;
   template <typename T>
   T apply_third_deriv(const T& x) const noexcept;
+};
+
+/*!
+ *  \ingroup MathFunctionsGroup
+ *  \brief 1D Gaussian \f$f = A \exp\left(-\frac{(x-x_0)^2}{w^2}\right)\f$
```

Comment:
this isn't 1D

Reply:
It might or might not be, so I just removed the "1D"